website/docs/r/namespace.html.md

```diff
@@ -79,6 +79,9 @@ The following arguments are supported:
 
 * `path` - (Required) The path of the namespace. Must not have a trailing `/`.
 
+* `custom_metadata` - (Optional) A map of arbitrary string to string valued user-provided metadata meant 
```

Comment:
nit: may want to update this doc to match the field description. If not should probably be `string to string values`?